packages/utils/lib/environment/parse.ts

```diff
@@ -349,6 +349,14 @@ export const ENVS = z.object({
     NANGO_ACTIVEMQ_PASSWORD: z.string().optional().default('admin'),
     NANGO_ACTIVEMQ_CONNECT_TIMEOUT_MS: z.coerce.number().optional().default(10_000),
 
+    // WEBHOOK DELIVERY CIRCUIT BREAKER
+    NANGO_WEBHOOK_TIMEOUT_MS: z.coerce.number().optional().default(20_000),
+    NANGO_WEBHOOK_RETRY_ATTEMPTS: z.coerce.number().optional().default(2),
+    NANGO_WEBHOOK_CIRCUIT_BREAKER_FAILURE_THRESHOLD: z.coerce.number().optional().default(5),
+    NANGO_WEBHOOK_CIRCUIT_BREAKER_WINDOW_SECS: z.coerce.number().optional().default(10),
+    NANGO_WEBHOOK_CIRCUIT_BREAKER_COOLDOWN_DURATION_SECS: z.coerce.number().optional().default(60),
+    NANGO_WEBHOOK_CIRCUIT_BREAKER_AUTO_RESET_SECS: z.coerce.number().optional().default(3600),
+
```

Comment:
I picked those values arbitrarily. I am happy to tweak them:
```
http requests timeout: 20s // currently 60s
retry attempt: 2 // currently 7
circuit breaker error rate threshold: 5 errors / 10 seconds // below that we accept to always make the requests
cooldown period before recovery: 60s // how long do we wait before attempting at sending webhook to an endpoint
auto reset: 1h // a way to auto clean circuit breaker if no requests are made after blocking. Since recovery is triggered by the following incoming webhooks, if there are no following webhook, the circuit breaker will be considered closed again after 1h
```